docs/how-to-use-relational-entity-storage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How to use relational entity storage
+slug: /how-to-use-relational-entity-storage
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Introduction
+
+Gravitino supports using `Relational Entity Storage` to store metadata after version `0.5.0`.  
+
+#### Target users
+`Relational Entity Storage` is mainly aimed at users who are accustomed to using `RDBMS` to store data or lack available a KV storage, and want to use Gravitino.  
+
+#### Advantages
+With `Relational Entity Storage`, you can quickly deploy Gravitino in a production environment and take advantage of relational
+storage to manage metadata.  
+Currently, you can use `MySQL` as the jdbc backend for `Relational Entity Storage`.
+
+## Steps for usage
+
+#### Prerequisites
+
++ MySQL 5.7 or 8.0
++ Gravitino distribution package
++ MySQL connector Jar (Should be compatible with your version of MySQL)
+
+#### Step 1: Get the initialization script
+
+You need `download` and `unzip` the distribution package firstly, please see: [How to install Gravitino](how-to-install.md).
+Then you can get the initialization script in the directory:
+```text
+${distribution_package_directory}/scripts/relational-entity-store/mysql/
```

Comment:
Is the directory name right?

Reply:
I found that the root directory names obtained in the release binary package (like `/gravitino-0.4.0-bin/scripts/`) and through compileDistribution (like `/distribution/package/scripts/`) are not consistent, so a variable is used to represent the two names.